examples/web/src/profiles/UseCreateProfile.tsx

```diff
@@ -1,7 +1,26 @@
-import { useCreateProfile } from '@lens-protocol/react-web';
+import { useCreateProfile, useProfilePrices } from '@lens-protocol/react-web';
+import { useEffect } from 'react';
 import toast from 'react-hot-toast';
 
 import { RequireConnectedWallet } from '../components/auth';
+import { formatAmount } from '../utils/formatAmount';
+
+function ProfilePrice() {
+  const { data, execute, loading, error } = useProfilePrices();
+
+  useEffect(() => {
+    void execute();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
the only problem is here, if `execute` is provided in useEffect deps, then the app is stuck in a never-ending loop.

Reply:
Can we brainstorm some options here? Seems a flaw in the way `useDeferredTask` is implemented.

Reply:
even if not perfect, I'd merge this branch as an example of onchain read hook, and keep it low profile, wdyt?